src/core/utils.js

```diff
@@ -535,6 +549,16 @@ export const validateParam = (param, isXml) => {
     let numberCheck = type === "number" && !validateNumber(value) // validateNumber returns undefined if the value is a number
     let integerCheck = type === "integer" && !validateInteger(value) // validateInteger returns undefined if the value is an integer
 
+    if (maxLength) {
```

Comment:
It's worth noting that `0` will fail this conditional, but I can't see why you'd ever want to set `minLength: 0` or `maxLength: 0`.

Reply:
Ye, I think that might play a bigger role on PR #3703 but here is not relevant

Reply:
I just discussed this with @webron - `maxLength: 0` is funky, but valid, so that case needs to be handled 😄  

`minLength: 0`, though, is not an issue, because nothing can be `length < 0`.